src/stores/x-stores.md

```diff
@@ -5,37 +5,37 @@ title: Stores
 
 In this section of the guide, you’ll learn how to set up a store hierarchy, create attributes, and manage the settings used by each store to calculate taxes, and currency rates,[![]({{ site.baseurl }}{% link images/images/tile-all-stores.png %})]({{ site.baseurl }}{% link stores/stores-menu.md %})
 
-## [All Stores]({{ site.baseurl }}{% link stores/stores-all-stores.md %})
+## All Stores
 
-Learn to manage the store hierarchy of websites, stores, and views. Manage taxes, and currency rates, and learn how attributes are used throughout the system. This section also links to a comprehensive configuration reference.
+Learn to manage the [store]({{ site.baseurl }}{% link stores/stores-all-stores.md %}) hierarchy of websites, stores, and views. Manage taxes, and currency rates, and learn how attributes are used throughout the system. This section also links to a comprehensive configuration reference.
 {: .indent0First}
 
 [![]({{ site.baseurl }}{% link images/images/tile-system.png %})]({{ site.baseurl }}{% link stores/configuration.md %})
 
-## [Configuration]({{ site.baseurl }}{% link stores/configuration.md %})
+## Configuration
 
-Our online Configuration Reference has field descriptions for every setting.
+Our online [Configuration]({{ site.baseurl }}{% link stores/configuration.md %}) Reference has field descriptions for every setting.
```

Comment:
Link should be for "Configuration Reference"

Reply:
**done**